app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -1760,7 +1766,7 @@ function BackgroundModelFilter({ parent }) {
   return (
     <OurDropdown
       options={backgroundOptions}
-      value={parent.state.backgroundId}
+      value={parseInt(parent.state.backgroundId)}
```

Comment:
Can we do this earlier? 
Do you know what is the source of non-int background id? If is cookies or even props we can parse it as soon as we get it.